controllers/storagecluster/reconcile.go

```diff
@@ -886,3 +890,14 @@ func getUnsupportedClientsCount(r *StorageClusterReconciler, namespace string) (
 
 	return count, nil
 }
+
+// Set dynamic default values in the StorageCluster CR
+func setDynamicDefaults(sc *ocsv1.StorageCluster) {
+
+	// Set default mon count value in the StorageCluster CR
+	if arbiterEnabled(sc) && sc.Spec.ManagedResources.CephCluster.MonCount != defaults.ArbiterModeMonCount {
```

Comment:
Whenever arbiter is enabled we must always set 5 mons, so I believe this simpler condition will ensure it.
```suggestion
	if arbiterEnabled(sc) {
```

Reply:
The reason I added that extra condition was to avoid resetting the same value again for the next reconcile, a small benefit but worth it I think?

Reply:
Ok, got it. I will update the condition.